docs/recipes/travis_build_stages.md

```diff
@@ -0,0 +1,90 @@
+# Using semantic-release with [Travis build stages](https://docs.travis-ci.com/user/build-stages/)
+
+![Travis Build Stages screenshot](https://cloud.githubusercontent.com/assets/3729517/25229553/0868909c-25d1-11e7-9263-b076fdef9288.gif)
+
+Build stages are a great way to run semantic release compared to running it in `after_success`:
+
+* The release stage can use Node 8, while the other stages can use other versions, even use a completely different language.
+* It allows you to easily release a project that is not even written in JavaScript, without impacting the job configuration of your test jobs in any way.
+* Instead of having to figure out a "build leader" to do the release and poll the Travis API to wait until all jobs passed, Travis can handle only running the release step after the other jobs passed.
```

Comment:
Here instead you can mentioned that using `node_js: node` will guarantee that the release will happen on that job.

Reply:
Maybe we should remove this sentence? I'm not sure the internal mechanics of the "non build stage" is relevant for the reader?

Reply:
I think it does matter because the Travis UI will show better what is going on, vs having the first job pending until all jobs completed. And it is more reliable than polling for sure.